Author: Modestas Vainius <modax@debian.org>
Description: improve detection of gpsd API
 Even if API major is bumped, it does not mean that this code won't build. So
 simply use >=.

--- a/marble/src/plugins/positionprovider/gpsd/GpsdConnection.cpp
+++ b/marble/src/plugins/positionprovider/gpsd/GpsdConnection.cpp
@@ -20,7 +20,7 @@
 {
     gps_data_t* data = m_gpsd.open();
     if ( data ) {
-#if defined( GPSD_API_MAJOR_VERSION ) && ( GPSD_API_MAJOR_VERSION == 3 || GPSD_API_MAJOR_VERSION == 4 ) && defined( WATCH_ENABLE ) 
+#if defined( GPSD_API_MAJOR_VERSION ) && ( GPSD_API_MAJOR_VERSION >= 3 ) && defined( WATCH_ENABLE )
         m_gpsd.stream( WATCH_ENABLE );
 #endif
         connect( &m_timer, SIGNAL( timeout() ), this, SLOT( update() ) );
@@ -32,7 +32,7 @@
 void GpsdConnection::update()
 {
     gps_data_t* data = 0;
-#if defined( GPSD_API_MAJOR_VERSION ) && ( GPSD_API_MAJOR_VERSION == 3 || GPSD_API_MAJOR_VERSION == 4 ) && defined( POLICY_SET )
+#if defined( GPSD_API_MAJOR_VERSION ) && ( GPSD_API_MAJOR_VERSION >= 3 ) && defined( POLICY_SET )
     while ((data = m_gpsd.poll()) && !(data->set & POLICY_SET)) {
         data = m_gpsd.poll();
     }
